_dev/apps/ui/src/components/product-feed/settings/summary/summary.vue

```diff
@@ -467,6 +476,8 @@ export default defineComponent({
   async mounted() {
     this.loadingData = true;
 
+    this.moduleNeedUpgradeForProductFilter = await this.$store.dispatch(`app/${AppActionsTypes.REQUEST_MODULE_NEED_UPGRADE}`, '1.73.0');
```

Comment:
Do we really need to make an API call when the version is already available on the app store?